README.md

```diff
@@ -1,2 +1,36 @@
-# bridge-relayer
-Relayer for Sora2 bridge
+<img alt="SORA logo" src="https://static.tildacdn.com/tild3664-3939-4236-b762-306663333564/sora_small.svg"/>
+
+# Overview
+
+Relayer for Sora2 bridges
+
+### Build
+
+```sh
+cargo b -r
+```
+# Run Relayers
```

Comment:
You can also add some info about how to generate `signer` seed and public key for relayer registration